index.html

```diff
@@ -940,9 +940,8 @@ <h3>Supporting algorithms</h3>
           Let |record:PressureRecord| be |observer|.{{PressureObserver/[[LastRecordMap]]}}[|source|].
         </li>
         <li>
-          If |record|.{{PressureRecord/[[State]]}} is not equal to |state| and [=contributing factors change threshold exceeded=]
-          given |factors| and |record|.{{PressureRecord/[[State]]}} returns true,
-          return true.
+          If |record|.{{PressureRecord/[[State]]}} is not equal to |state| and [=change in contributing factors is substantial=]
```

Comment:
"and [=change in contributing factors is substantial=]"
Why do we need this? it is covered by the next "If" statement

Reply:
The next if statement:

>If record.[[[Factors]]](https://www.w3.org/TR/compute-pressure/#dfn-factors) and factors are not [supersets](https://infra.spec.whatwg.org/#set-superset) of each other, return true.

... checks if one ordered set (~list) is a superset of another ordered set (~list):

AFAICT that if statement does not call into "change in contributing factors is substantial". We need to call into it from somewhere. Based on my reading of the spec, this is the place to do that. If there's a better place, please let me know and I'll update the PR.

Reply:
Yeah, this looks correct to me